apiserver/facades/agent/upgradeseries/upgradeseries.go

```diff
@@ -135,7 +135,7 @@ func (a *API) CurrentSeries(args params.Entities) (params.StringResults, error)
 			results[i].Error = apiservererrors.ServerError(err)
 			continue
 		}
-		series, err := coreseries.GetSeriesFromChannel(machine.Base().OS, machine.Base().Channel)
+		series, err := corebase.GetSeriesFromChannel(machine.Base().OS, machine.Base().Channel)
```

Comment:
We have plans to remove this and use base directly I assume. So this is just an interim step.

Reply:
:+1: